__tests__/users/email.test.ts

```diff
@@ -0,0 +1,33 @@
+import sendEmail from "../../src/utils/email";
+import userModel from "../../models/user.model";
+
+describe("sendEmail", () => {
+  it("should send email", async () => {
+    jest.setTimeout(30000);
+    const token = new userModel().generateAuthToken();
+    const email = "test@test.com";
+    const res = await sendEmail(email, token);
+    console.log(res);
+    expect(res).not.toBeNull();
+    expect(res).not.toBeUndefined();
+    expect(res).toBe("Mail send!");
+  });
+});
+
+describe("error", () => {
+  it("tests error with async/await", async () => {
+    try {
+      jest.setTimeout(30000);
+      const token = new userModel().generateAuthToken();
+      const email = "test@test.com";
+      const res = await sendEmail(email, token);
+      console.log(res);
```

Comment:
spróbuj usunąć console.log i zobacz czy dalej będzie ostrzeżenie w trakcie wykonywania testu

Reply:
Jak mówiłam ostatnio na spotkaniu - próbowałam. To nic nie daje.